src/index.ts

```diff
@@ -7,12 +7,14 @@ async function run(): Promise<void> {
         const slackToken = core.getInput("slack-token", { required: true });
         const channel = core.getInput("channel", { required: true });
         const message = core.getInput("message", { required: true });
+        const markdown = core.getInput("mrkdwn", {required: false });
```

Comment:
🤔 I wonder if people will misspell `mrkdwn`, would calling the input "markdown" be incorrect?

Reply:
+ 1 for naming it `markdown` instead of `mrkdwn`